docs/doc/15-sql-functions/04-array-functions/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Array Functions'
+---
+
+Array Functions
+
+## Syntax
+
+| Function    | Description             |  Example | Result
+| ----------- | ------------------ |  ---------- | --------|
+| `get(array1, index)`  | Get item from an array by index     |  `get([1,2], 2)`         |  2
```

Comment:
`get(array, index)`

Reply:
![image](https://user-images.githubusercontent.com/172204/218308787-afffa41a-2dcf-43cd-b008-0ddd6e0068d7.png)
Why databend index start from 1 not 0?

Reply:
Almost `> 90%` of databases use 1 based index in SQL as SQL-99 standard.

Reply:
Right, but not a good experience(Not many people are familiar with SQL-99 standard).
We should explicitly `(1-based)` it like duckdb:
https://duckdb.org/docs/sql/functions/nested#list-functions